MBBSEmu/Program.cs

```diff
@@ -83,6 +84,11 @@ public class Program
         /// </summary>
         private readonly List<ModuleConfiguration> _moduleConfigurations = new List<ModuleConfiguration>();
 
+        /// <summary>
+        ///     Channel Dictionary
+        /// </summary>
+        private readonly PointerDictionary<SessionBase> _channelDictionary = new PointerDictionary<SessionBase>();
```

Comment:
This will be a second dictionary not part of the one in DI. We should remove this.